tatanka/spec/intro.md

```diff
@@ -0,0 +1,93 @@
+# Introduction to Tatanka Mesh
+
+Tatanka Mesh (TM, or just "the mesh") is a multi-blockchain mesh network offering a suite
```

Comment:
Could you elaborate on what "multi-blockchain" refers to here? It seems like Tatanka is a generic mesh communication protocol which could be used for any purpose. The initial use case for Tatanka may be for blockchain related services, but is anything about it actually blockchain specific?

Reply:
Tatanka is indeed designed to be generic, and non-blockchain apps could be implemented. What makes the mesh "multi-blockchain" is 1) the blockchain oracle services, 2) the HTLC auditing service, and to a lesser extent 3) the bonding system.